api/v1beta1/foundationdbcluster_types_test.go

```diff
@@ -496,7 +496,8 @@ func TestParsingClusterStatusWithSixOneCluster(t *testing.T) {
 						"processid":   "c813e585043a7ab55a4905f465c4aa52",
 						"zoneid":      "sample-cluster-storage-3",
 					},
-					Version: "6.1.12",
+					Version:       "6.1.12",
+					UptimeSeconds: 160.009,
```

Comment:
Is this uptimeSeconds value going to change between tests? Or if we add other settings/processing in our methods? I want to make sure this isn't going to be a brittle test case.

Reply:
Much of the contents of the status can change if we take a fresh status JSON, including the process IDs. However, these test cases are based off of status JSON output from live clusters, which we keep in the repository, and we shouldn't have any need to update the JSON file for an existing test case. When we add support for new versions, or possibly some new features or configurations, we'll add new test cases and create new JSON files to support those test cases.